tools/solidity-coverage-example/README.md

```diff
@@ -0,0 +1,34 @@
+# Sample Hardhat Project
+
+This project demonstrates a basic Hardhat use case with Hedera. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
```

Comment:
The goal here is to show off solidity-coverage not hardhat, so you should update this description